lib/bot/cmds/thanks.js

```diff
@@ -1,57 +1,85 @@
 'use strict';
 
-const Utils = require('../../../lib/utils/Utils'),
-      HttpWrap = require('../../../lib/utils/HttpWrap'),
-      TextLib = require('../../../lib/utils/TextLib');
+const Utils = require('../../../lib/utils/Utils');
+const HttpWrap = require('../../../lib/utils/HttpWrap');
+const TextLib = require('../../../lib/utils/TextLib');
+const dedent = require('dedent');
```

Comment:
at first i not understand why you not use `single-var-pattern`, but after read this http://danhough.com/blog/single-var-pattern-rant/  i understand this